### PR TITLE
docs: update the docs of Limits and Data Types(#1214)

### LINKED ISSUE #1214: docs: update the docs of Limits、Compile Methods and Data Types
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/stoneatom/stonedb/blob/stonedb-5.7-dev/CONTRIBUTING.md).

### Description

update some docs：
- Limits
- Compile Methods
- Data Types

### Self-service

- [X] I'd be willing to address this documentation request myself.

### PR DESCRIPTION
**Pull request by Nliver**
## Summary about this PR

Issue Number: close #1214

update the docs of Limits and Data Types.

## Tests Check List

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [X] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [X] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [X] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

**Review by haitaoguan — APPROVED** [review]

**Review by haitaoguan — APPROVED** [review]

File: Docs/05-SQL-reference/data-types.md [hunk 1]
```diff
@@ -16,45 +15,42 @@ The following table lists the data types supported by StoneDB.
...
+| --- | --- | --- | --- | --- |
+| TINYINT | 1 | Yes | -128 | 127 |
+|  |  | No | 0 | 127 |
+| SMALLINT | 2 | Yes | -32768 | 32767 |
+|  |  | No | 0 | 32767 |
+| MEDIUMINT | 3 | Yes | -8388608 | 8388607 |
+|  |  | No | 0 | 8388607 |
+| INT | 4 | Yes | -2147483647 | 2147483647 |
+|  |  | unsigned | 0 | 2147483647 |
+| BIGINT | 8 | Yes | -9223372036854775808 | 9223372036854775807 |
```
Comment (haitaoguan) [inline Docs/05-SQL-reference/data-types.md:31]: | BIGINT | 8 | Yes | -9223372036854775806 | 9223372036854775807 |

File: Docs/05-SQL-reference/data-types.md [hunk 2]
```diff
@@ -16,45 +15,42 @@ The following table lists the data types supported by StoneDB.
...
+| **Data type** | **Bytes of storage** | **Signed** | **Min. value** | **Max. value** |
+| --- | --- | --- | --- | --- |
+| TINYINT | 1 | Yes | -128 | 127 |
+|  |  | No | 0 | 127 |
+| SMALLINT | 2 | Yes | -32768 | 32767 |
+|  |  | No | 0 | 32767 |
+| MEDIUMINT | 3 | Yes | -8388608 | 8388607 |
+|  |  | No | 0 | 8388607 |
+| INT | 4 | Yes | -2147483647 | 2147483647 |
+|  |  | unsigned | 0 | 2147483647 |
```
Comment (haitaoguan) [inline Docs/05-SQL-reference/data-types.md:30]: |  |  | No | 0 | 2147483647 |

File: Docs/05-SQL-reference/data-types.md [hunk 3]
```diff
@@ -16,45 +15,42 @@ The following table lists the data types supported by StoneDB.
...
+| BIGINT | 8 | Yes | -9223372036854775808 | 9223372036854775807 |
+|  |  | unsigned | 0 | 18446744073709551615 |
```
Comment (haitaoguan) [inline Docs/05-SQL-reference/data-types.md:32]: @haitaoguan	Reply...
|  |  | No | 0 | 9223372036854775807 |

File: website/i18n/zh/docusaurus-plugin-content-docs/current/05-SQL-reference/data-types.md [hunk 4]
```diff
@@ -16,19 +16,22 @@ StoneDB 支持如下的数据类型。
...
+| --- | --- | --- | --- | --- |
+| TINYINT | 1 | 有符号 | -128 | 127 |
+|  |  | 无符号 | 0 | 127 |
+| SMALLINT | 2 | 有符号 | -32768 | 32767 |
+|  |  | 无符号 | 0 | 32767 |
+| MEDIUMINT | 3 | 有符号 | -8388608 | 8388607 |
+|  |  | 无符号 | 0 | 8388607 |
+| INT | 4 | 有符号 | -2147483647 | 2147483647 |
+|  |  | 无符号 | 0 | 2147483647 |
+| BIGINT | 8 | 有符号 | -9223372036854775808 | 9223372036854775807 |
```
Comment (haitaoguan) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/05-SQL-reference/data-types.md:31]: | BIGINT | 8 | 有符号 | -9223372036854775806 | 9223372036854775807 |

File: website/i18n/zh/docusaurus-plugin-content-docs/current/05-SQL-reference/data-types.md [hunk 5]
```diff
@@ -16,19 +16,22 @@ StoneDB 支持如下的数据类型。
...
+|  |  | 无符号 | 0 | 18446744073709551615 |
```
Comment (haitaoguan) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/05-SQL-reference/data-types.md:32]: |  |  | 无符号 | 0 | 9223372036854775807 |

**Review by RingsC — APPROVED** [review]

**Pull request merged**